`install_rstudio.sh` fails to run

Hello,

We are facing an issue while building a docker image which has following snippet
```
RUN git clone https://github.com/rocker-org/rocker-versioned2.git /root/rocker/
RUN mv /root/rocker/scripts /rocker_scripts
RUN /rocker_scripts/install_rstudio.sh
RUN /rocker_scripts/install_pandoc.sh
RUN rm -rf /rocker_scripts /root/rocker
```
The error itself does not reveal much as to why this is failing, only that it failed to run **`install_rstudio.sh`** script successfully
```
2021-10-11 18:54:41 (28.5 MB/s) - ‘/tmp/s6-overlay-amd64.tar.gz’ saved [1896116/1896116]

The command '/bin/sh -c /rocker_scripts/install_rstudio.sh' returned a non-zero code: 1

Error: Process completed with exit code 1.
```

We are leveraging repo [htappen/r-google-cloud-ai](https://github.com/htappen/r-google-cloud-ai/blob/main/Docker/Dockerfile). Earlier there were no issues when the docker was built.

When run independently, the script runs successfully 
```
sudo /bin/sh -c /rocker_scripts/install_rstudio.sh
```

Any help on this is appreciated! 

Thanks!
Heramb

@erichall87 @rajsinghgm

--- a/scripts/install_rstudio.sh
+++ b/scripts/install_rstudio.sh
@@ -45,7 +45,7 @@ fi
 RSTUDIO_BASE_URL=https://download2.rstudio.org/server
 
 if [ -z "$RSTUDIO_VERSION_ARG" ] || [ "$RSTUDIO_VERSION_ARG" = "latest" ]; then
-    DOWNLOAD_VERSION=$(wget -qO - https://rstudio.com/products/rstudio/download-server/debian-ubuntu/ | grep -oP "(?<=rstudio-server-)[0-9]+\.[0-9]+\.[0-9]+%2B[0-9]+" -m 1)
+    DOWNLOAD_VERSION=$(wget -qO - https://rstudio.com/products/rstudio/download-server/debian-ubuntu/ | grep -oP "(?<=rstudio-server-)[0-9]+\.[0-9]+\.[0-9]+-[0-9]+" -m 1)
 elif [ "$RSTUDIO_VERSION_ARG" = "preview" ]; then
     DOWNLOAD_VERSION=$(wget -qO - https://rstudio.com/products/rstudio/download/preview/ | grep -oP "(?<=rstudio-server-)[0-9]+\.[0-9]+\.[0-9]+-preview%2B[0-9]+" -m 1 ||
       wget -qO - https://rstudio.com/products/rstudio/download/preview/ | grep -oP "(?<=rstudio-server-)[0-9]+\.[0-9]+\.[0-9]+%2B[0-9]+" -m 1)

--- a/scripts/userconf.sh
+++ b/scripts/userconf.sh
@@ -103,7 +103,7 @@ if [ "$TZ" !=  "Etc/UTC" ]
 fi
 
 ## Set our dynamic variables in Renviron.site to be reflected by RStudio
-exclude_vars="HOME PASSWORD"
+exclude_vars="HOME PASSWORD RSTUDIO_VERSION"
 for file in /var/run/s6/container_environment/*
 do
   sed -i "/^${file##*/}=/d" ${R_HOME}/etc/Renviron.site

